### PR TITLE
Added dockerfile, actions and minor changes

### PR DESCRIPTION
**Pull request by brgvgns**

### CODE REVIEW

File: incoming-sms.php [hunk 1]
```diff
@@ -18,11 +19,13 @@
...
 
+$request = json_decode(file_get_contents('php://input'), true);
+
 $account_id = $_ENV['ACCOUNT_ID']; // string | ID of the account
 $data = array(
-    'from' => $_ENV['FROM'],
+    'from' => $_ENV['FREECLIMB_NUMBER'],
     //FC Number
-    'to' => $_ENV['TO'],
+    'to' => $request['from'],
```
Comment (ngarg-vail) [inline incoming-sms.php:36]: This is a little confusing. based on the file name (incoming sms) this should handle incoming sms, but the contents indicate we're sending SMS to a destination number from the account holders freeclimb number.

Coupled with`$request` being driven by a file input that doesn't exist. 
The simpler approach would be to retain the environment variables to handle the from/to numbers to send the message

File: index.php [hunk 2]
```diff
@@ -47,24 +47,24 @@
...
+			<p>View an example PerCL JSON response to FreeClimb at <a href="http://localhost:3000/incoming-sms.php" target="_blank">http://127.0.0.1:3000/incoming-sms.php</a></p>
 			<p>Your next step is to use ngrok to proxy HTTP traffic to this local web server, and then link the ngrok proxy url to your FreeClimb application </p>
 			<ol>
-				<li>Using ngrok, configure the dynamic url to proxy to http://127.0.0.1:8080 using command <pre><code>ngrok http 8080</code></pre></li>
-				<li>Using the FreeClimb Dashboard or API, set your FreeClimb Application Voice Url to the dynamic endpoint NGROK generated <pre><code>e.g. https://{ngrok-generated-url}/incoming-sms.php</code></pre></li>
+				<li>Using ngrok, configure the dynamic url to proxy to http://127.0.0.1:3000 using command <pre><code>ngrok http 3000</code></pre></li>
+				<li>Using the FreeClimb Dashboard or API, set your FreeClimb Application SMS Url to the dynamic endpoint NGROK generated <pre><code>e.g. https://{ngrok-generated-url}/incoming-sms.php</code></pre></li>
 			</ol>
 		</div>
 	</body>
```
Comment (ngarg-vail) [inline index.php:70]: looks like we have some drift. a README file is sufficient, I don't think having an additional index.php file documenting the same is necessary

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

File: incoming-sms.php [hunk 3]
```diff
@@ -18,11 +19,13 @@
     $config
 );
 
+$request = json_decode(file_get_contents('php://input'), true);
```
Comment (brgvgns) [inline incoming-sms.php:30]: Parses the FreeClimb Incoming Webhook Request through raw http post data

Comment (ngarg-vail) [reply incoming-sms.php:30]: @brgvgnsadd this as a code comment to the file above the `$request` line

Comment (brgvgns) [reply incoming-sms.php:30]: @ngarg-vail should be in now!

**Review by ngarg-vail — APPROVED** [review]

**Pull request merged**